runtime/binding-kafka-grpc/src/main/java/io/aklivity/zilla/runtime/binding/kafka/grpc/internal/stream/KafkaGrpcFetchHeaderHelper.java

```diff
@@ -90,11 +114,26 @@ private boolean dispatch(
     {
         final OctetsFW name = header.name();
         final OctetsFW value = header.value();
+        final int offset = name.offset();
+        final int limit = name.limit();
+
+        name.buffer().getBytes(offset, HEADER_META_PREFIX);
+        name.buffer().getBytes(limit - BIN_SUFFIX.length, HEADER_BIN_SUFFIX);
+
         final Consumer<OctetsFW> visitor = visitors.get(name);
         if (visitor != null)
         {
             visitor.accept(value);
         }
+        else if (Arrays.equals(META_PREFIX, HEADER_META_PREFIX))
+        {
+            final GrpcType type = Arrays.equals(BIN_SUFFIX, HEADER_BIN_SUFFIX) ? BASE64 : TEXT;
```

Comment:
I checked the spec scripts and don't yet see any usage of this approach, so please modify the scripts to all use both `text` and `binary` metadata instead of just `text`.